src/platform/generic/_locales/de/messages.json

```diff
@@ -1176,5 +1176,117 @@
+    },    
+    "LabelSearchRecommendationRows6"                 : {
+        "message"    : "6",
+        "description": "Number of results that should be displayed for password recommendations."
+    },    
+    "LabelSearchRecommendationRows7"                 : {
+        "message"    : "7",
+        "description": "Number of results that should be displayed for password recommendations."
+    },    
+    "LabelSearchRecommendationRows8"                 : {
+        "message"    : "8",
+        "description": "Number of results that should be displayed for password recommendations."
+    },    
+    "LabelSearchRecommendationRows9"                 : {
+        "message"    : "9",
+        "description": "Number of results that should be displayed for password recommendations."
+    },    
+    "LabelSearchRecommendationRows10"                 : {
+        "message"    : "10",
+        "description": "Number of results that should be displayed for password recommendations."
+    },    
+    "LabelSearchRecommendationRows11"                 : {
+        "message"    : "11",
+        "description": "Number of results that should be displayed for password recommendations."
+    },    
+    "LabelSearchRecommendationRows12"                 : {
+        "message"    : "12",
+        "description": "Number of results that should be displayed for password recommendations."
+    },    
+    "LabelSearchRecommendationRows13"                 : {
+        "message"    : "13",
+        "description": "Number of results that should be displayed for password recommendations."
+    },    
+    "LabelSearchRecommendationRows14"                 : {
+        "message"    : "14",
+        "description": "Number of results that should be displayed for password recommendations."
+    },    
+    "LabelSearchRecommendationRows15"                 : {
+        "message"    : "15",
+        "description": "Number of results that should be displayed for password recommendations."
+    },    
+    "LabelSearchRecommendationRows16"                 : {
+        "message"    : "16",
+        "description": "Number of results that should be displayed for password recommendations."
+    },    
+    "LabelSearchRecommendationRows17"                 : {
+        "message"    : "17",
+        "description": "Number of results that should be displayed for password recommendations."
+    },    
+    "LabelSearchRecommendationRows18"                 : {
+        "message"    : "18",
+        "description": "Number of results that should be displayed for password recommendations."
+    },    
+    "LabelSearchRecommendationRows19"                 : {
+        "message"    : "19",
+        "description": "Number of results that should be displayed for password recommendations."
+    },    
+    "LabelSearchRecommendationRows20"                 : {
+        "message"    : "20",
+        "description": "Number of results that should be displayed for password recommendations."
```

Comment:
We can't do that to our translators. See below for suggestion.

Reply:
I agree an dI was also not happy with my implementation. I tried befor with the id a slabel but this caused some errors in the console log. So I used this workaround. 

But your appreach is of course better. I updated my implementation according your instructions.